.gitignore

```diff
@@ -0,0 +1 @@
+*.retry
```

Comment:
I will remove this. This was supposed to prevent from getting these *.retry files, that get generated if an ansible script fails, from being commited by mistake.